### PR TITLE
new page

### PR DESCRIPTION
**Pull request by blam-codes**
The page is not completely and I did not have time to check to see if the indentation was perfect but I think we can discuss if I am using the right elements and  some general notes on code during tonight's meeting.

### CODE REVIEW

File: opentext-page.html [hunk 1]
```diff
@@ -0,0 +1,499 @@
...
+            .container-3 {
+                margin-top: 70px;
+                width: 25%;
+                display: inline;
+                float: left;              
+            }
+            .column {
+            float: left;
+            width: 20%;
+        }s
```
Comment (dlamdevelops) [inline opentext-page.html:44]: There's an extra "s" here?

Comment (dlamdevelops) [inline opentext-page.html:44]: Indentation issue with the } ... this needs to align; and it also throws off the alignment of the css selectors on lines 45-71

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Please go over indentation and extra lines/spaces.

**Comment by blam-codes** [comment]
Added to sections to page and corrected indentation issues.

File: opentext-page.html [hunk 2]
```diff
@@ -0,0 +1,482 @@
...
+                    </ul>
+                </div>
+            </div>
+        </div>
+        <div class="slide-show">
+            <!-- div has two columns will add second column when learn about slideshows-->
+            <img src="#img1" alt="banner for voyager fund">
+            <h2>OpenText launches Voyager fund</h2>
+            <p>1M donation to help fight food insecurity</p>
+            <button type="submit">Learn more</button>
```
Comment (dlamdevelops) [inline opentext-page.html:177]: These three lines exist in a column, separate from image above.
Please apply this to all slides.

File: opentext-page.html [hunk 3]
```diff
@@ -0,0 +1,482 @@
...
+                    <!-- looks like buttons-->
+                    <span class="span">Developer</span>
+                </div>
+            </div>
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Contact us</span>
```
Comment (dlamdevelops) [inline opentext-page.html:223]: same issue as above, change this across the board for class names...
should never be their element name

File: opentext-page.html [hunk 4]
```diff
@@ -0,0 +1,482 @@
...
+            <div class="row">
+                <div class="column">
+                    <ul>
+                        <li><a href="#Business">Business Network Cloud</a></li>
+                        <li><a href="#Security">Security & Protection Cloud</a></li>
+                    </ul>
+                </div>
+            </div>
+        </div>    
+        <div class="client">
```
Comment (dlamdevelops) [inline opentext-page.html:260]: Requires also container class here, and applies to all other cases below with a div w/ client class.

File: opentext-page.html [hunk 5]
```diff
@@ -0,0 +1,482 @@
...
+        </div>
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Vodafone">Vodafone logo</a>                               
+                </div>
+            </div>
+            <button type="submit">Read more Sucess stories</button>
+        </div>
+        <div class="blog">
```
Comment (dlamdevelops) [inline opentext-page.html:317]: Requires a container class.... same issue as client divs above.

File: opentext-page.html [hunk 6]
```diff
@@ -0,0 +1,482 @@
...
+            <div class="row">
+                <div class="column">
+                    <a href="#News">Latest news and insights</a>
+                        <!-- no borders with an arrow -->
+                        <button type="submit">See All</button>
+                    </a>
+                </div>
+            </div>
+        </div>
+        <div class="blog">
```
Comment (dlamdevelops) [inline opentext-page.html:327]: same issue... you keep dropping the use of container class
fix all instances.... stopping now from pointing all of them out...

File: opentext-page.html [hunk 7]
```diff
@@ -0,0 +1,482 @@
...
+            <div class="row">
+                <div class="column">
+                    <!--has arrow --> 
+                    <a href="#Name">What is in a name?</a>                              
+                </div>
+            </div>
+        </div>
+        <footer class="footer">
```
Comment (dlamdevelops) [inline opentext-page.html:359]: same issue with naming a class "footer" on a footer element

Comment (dlamdevelops) [inline opentext-page.html:359]: requires a container class

Comment (dlamdevelops) [inline opentext-page.html:359]: The `<footer>` tag is misused.
It's not a tag this is used over and over for footer columns.
Instead, this is the format.  Please re-do this entire footer section.

```
<footer class="container">
    <div class="row">
       <div class="column">
           ... links and stuff here
       </div>
       <div class="column">
           ... links and stuff here
       </div>
       <div class="column">
           ... links and stuff here
       </div>
   </div>
</footer>
```

File: opentext-page.html [hunk 8]
```diff
@@ -0,0 +1,482 @@
...
+                        <li><a href="#Investors">Investors</a><li>
+                        <li><a href="#Office">Office locations</a><li>
+                        <li><a href="#Governance">Corporate governance</a><li></li>
+                        <li><a href="#Citizenship">Corporate citizenship</a><li></li>
+                    </ul>
+                </div>
+            </div>
+        </div>
+        </footer>
+        <div class="contact">
```
Comment (dlamdevelops) [inline opentext-page.html:445]: same issue with missing container class
fix all instances of this

File: opentext-page.html [hunk 9]
```diff
@@ -0,0 +1,482 @@
...
+            <div class="row">
+                <div class="col-5">
+                    <!-- socials with css all fit into the same column -->
+                    <a href="#twitter">twitter</a>
+                    <a href="#linkedln">linkedln</a>
+                    <a href="#youtube">youtube</a>
+                </div>
+            </div>
+        </div>
+        <div>
```
Comment (dlamdevelops) [inline opentext-page.html:470]: Missing 
container > row > column structure

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Please see changes and log the exact error type/issue into your list...

File: opentext-page.html (hunk 2 above)
Comment (blam-codes) [reply opentext-page.html:177]: Added a note in  my code will learn how to do slides when we get to css.

File: opentext-page.html [hunk 10]
```diff
@@ -0,0 +1,482 @@
...
+        </div>
+        <div class="container-2">
+            <div class="row">
+                <div class="column">
+                    <!-- looks like buttons-->
+                    <span class="span">Contact us</span>
+                </div>
+            </div>
+        </div>
+        <div>
```
[the remark this answers is not in the document]
Comment (blam-codes) [reply opentext-page.html:227]: I don’t understand it is sitting in a class of container-2 which is in a row and column layout.

File: opentext-page.html (hunk 6 above)
Comment (blam-codes) [reply opentext-page.html:327]: What do you mean by container class?

File: opentext-page.html (hunk 10 above)
[the remark this answers is not in the document]
Comment (dlamdevelops) [reply opentext-page.html:227]: This is not sitting in a div with a class of container-2.

File: opentext-page.html (hunk 6 above)
Comment (dlamdevelops) [reply opentext-page.html:327]: Line 86 is an example.
You keep dropping these and instead either doing something like line 327, or an HTML element tag like `<footer>`.
In reality, you only ever truly have one container class, but for the sake of consistency, I am asking you to add them here.  In your next page, we will go over the structure for container > row > column again.

**Review by dlamdevelops — CHANGES_REQUESTED** [review]
Please see comments to your questions....

**Comment by blam-codes** [comment]
Hi David,

I am confused as to what you mean by "not sitting in a div with a class
container-2". With the footer can I use the same format I used for the
footer for Flux Academy?

Best,

Ber

File: opentext-page.html [hunk 11]
```diff
@@ -0,0 +1,482 @@
...
+            </div>
+        </div>    
+        <div class="client">
+            <div class="row">
+                <div class="col-4">
+                    <a href="#Accelerate">Accelerate customer communication</a>                                
+                </div>
+            </div>
+        </div>
+        <div class="client">
```
Comment (blam-codes) [inline opentext-page.html:267]: put container class, make sure it is first class then client

File: opentext-page.html [hunk 12]
```diff
@@ -0,0 +1,501 @@
...
+            .col {
+                width: 16.6%;
+            }
+            a {
+                display: block;
+            }
+            .slide-show {
+                width: 50%;
+            }            
+            img {
```
Comment (dlamdevelops) [inline opentext-page.html:28]: Although there is nothing technically wrong here (you can style HTML element tags directly like img, p, a), this selector targets ALL images in your html file.  The styles you have within this selector are width of 250px and float of left, which seems specific and wouldn't apply to all images.

In the future, unless you are targeting an entire group of tags, it's better to use classes or id's as the selector. 

No need to fix this here.  Just letting you know...

**Review by dlamdevelops — APPROVED** [review]
Approved.  Please squash and let me know when you have done so.  We will come back to refactor this at a later date to practice moving code around.  Please see the one comment I have about line 28 (styling element tags directly).

**Comment by blam-codes** [comment]
File has been squashed.

Ber

**Pull request merged**